includes/class-wc-payments-utils.php

```diff
@@ -992,4 +992,31 @@ public static function enqueue_style( $handle, $path = '', $deps = [], $version
 		}
 		wp_enqueue_style( $handle );
 	}
+
+	/**
+	 * Returns language data: english name and native name
+	 *
+	 * @param string $language Language code.
+	 *
+	 * @return array
+	 */
+	public static function get_language_data( $language ) {
+		require_once ABSPATH . 'wp-admin/includes/translation-install.php';
+
+		$transaltions = wp_get_available_translations();
+
+		if ( isset( $transaltions[ $language ] ) ) {
+			return [
+				'code'         => $language,
+				'english_name' => $transaltions[ $language ]['english_name'] ?? '',
```

Comment:
I was wondering if it's better to return the default values as fallbacks rather than empty strings.

Reply:
It's completely logical. Yet, I believe it's more beneficial to return the `$language` instead of the default `English (United States)` to avoid any potential confusion among the merchants.